src/components/issues/details/IssueComment.js

```diff
@@ -14,6 +14,10 @@ const IssueComment = (props: Props) => {
   const avatar = get('user.avatar_url', props.context);
   const { created_at: createdAt } = props.context;
 
+  const image = props => {
+    return <img {...props} style={{ maxWidth: '100%' }} />;
```

Comment:
rest here makes it unclear for someone else on which props are received here. it is also error prone since you are not protecting it from receiving unwanted props